service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/SendDocumentControllerTest.java

```diff
@@ -56,7 +56,6 @@
 @OverrideAutoConfiguration(enabled = true)
 class SendDocumentControllerTest extends AbstractControllerTest {
 
```

Comment:
Just seen in this test that time is mocked to return a specifc time, don't think that should be necessary.